tests/st/bgp/test_global_peers.py

```diff
@@ -40,8 +39,8 @@ def test_global_peers(self):
                         additional_docker_options=ADDITIONAL_DOCKER_OPTIONS,
```

Comment:
The ADDITIONAL_DOCKER_OPTIONS have been made part of the common code and removed from bgp.

Can now use CLUSTER_STORE_DOCKER_OPTIONS in docker_host.py